### PR TITLE
Task 11 &12

### PR DESCRIPTION
**Pull request by Iadolphe007**
Review and merge

### CODE REVIEW

File: api/v1/views/places_reviews.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+    if 'user_id' not in data:
+        return make_response(jsonify({"error": "Missing user_id"}), 400)
+    user_id = data['user_id']
+    user = storage.get(User, user_id)
+    if user is None:
+        abort(404)
+    if 'text' not in data:
+        return make_response(jsonify({'error': 'Missing text'}), 400)
+
+    revire = Review(**data)
```
Comment (joelmuhoho) [inline api/v1/views/places_reviews.py]: this should be  review, not revire

File: api/v1/views/places_reviews.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+
+@app_views.route('/places/<place_id>/reviews', methods=['GET'],
+                 strict_slashes=False)
+def get_review_by_place(place_id):
+    """get review by place"""
+    place = storage.get(Place, place_id)
+    if place is None:
+        abort(404)
+    reviews = [review.to_dict() for review in place.reviews)
```
Comment (joelmuhoho) [inline api/v1/views/places_reviews.py]: reviews should be a list... check the closing bracket you used ) instead of ]

**Review by joelmuhoho — CHANGES_REQUESTED** [review]
kindly check on the two issues.

**Comment by Iadolphe007** [comment]
all issues fixed review again

**Review by joelmuhoho — APPROVED** [review]

**Pull request merged**